test/CMakeLists.txt

```diff
@@ -361,7 +361,7 @@ if( iodaconv_bufr_ENABLED )
   )
 endif()
 
-if (iodaconv_satbias_ENABLED )
+if (iodaconv_varbc_gsi_ENABLED )
```

Comment:
understand the GSI name here as it is specific to your predictor choices, but maybe NOAA ?